### PR TITLE
Register page - Issue Solved

### PR DESCRIPTION
**Pull request by RaviprasathKJ**
### **This pull request solves the following changes**
* Fixes Issue - [17 ](https://github.com/cittakshashila/frontend/issues/17)

###  **Checklist**
- [x] Replaced the ul with input tag .
- [x] Used the useState in the input tag .
- [x] Made the input responsive .
- [x] validated using zod and react hook forms and used toastify to display errors
- [ ] responsiveness 

ptal 🤞 @BharathSanjeeviT @MKMukeshkannan .

### CODE REVIEW

**Comment by RaviprasathKJ** [comment]
Added the underline .

File: src/app/register/page.tsx [hunk 1]
```diff
@@ -26,33 +32,33 @@ const Register = () => {
                         src={TakshashilaText}
                     />
                </div>
-              <div className="ml-5 text-xl md:text-2xl lg:text-3xl font-passport">
-                    <div className="ml-5 my-[1.5rem] sm:my-[30px] font-passport">
-                        <u >name</u>
+              <div className="ml-5 text-xl md:text-2xl lg:text-3xl font-passport flex flex-col justify-around  gap-[1rem] sm:gap-[2.5rem] flex-start ">
```
Comment (MKMukeshkannan) [inline src/app/register/page.tsx:35]: - The page is a registration form, So kindly use form tag

File: src/app/register/page.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
-"use client"
+"use client" ;
+import {useState} from "react";
 import Image from "next/image";
 import useHighlightScroll from "@/hooks/useHighlightScroll";
 import TakshashilaText from "../../../public/takshashila-text-2024.svg";
```
Comment (MKMukeshkannan) [inline src/app/register/page.tsx:5]: - just `"/takshasila-text-2024.svg"` will import it from public folder.

File: src/app/register/page.tsx [hunk 3]
```diff
@@ -11,6 +12,11 @@ const Register = () => {
   const defaultStyle =
     "no-underline	text-cream border border-transparent transition-all duration-200 p-[3px] rounded";
 
+    const [name,setName] : string = useState("");
```
Comment (MKMukeshkannan) [inline src/app/register/page.tsx:15]: - I suggest you to use `Zod` for form validation and `react-hook-form` for form submission

**Review by MKMukeshkannan — CHANGES_REQUESTED** [review]
- use zod for form validation
- react-hook-form for submission
- use from tag and button with type `submit`
- Btw, the current commit is have _build issues_, kindly fix it
Screenshot 2024-02-11 at 5 53 58 PM

**Comment by RaviprasathKJ** [comment]
@MKMukeshkannan 
Can please reviee my code , I have done the changes that you have asked .

**Pull request closed without merging**